### PR TITLE
Updater

### PR DESCRIPTION
**Pull request by deanshub**

### CODE REVIEW

File: src/updater.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+interface Release {
+    body: string | undefined
+    published_at: string | undefined
+    created_at: string | undefined
+    tag_name: string | undefined
+    name: string | undefined
+}
+
+async function checkForUpdate(bot: Telegraf<Context>) {
+    // TODO: Maybe get the url from origin remote and fallback to deanshub/downloader?
```
Comment (deanshub) [inline src/updater.ts:80]: @Schniz are you using a fork? do you need this?

Comment (Schniz) [reply src/updater.ts:80]: You can add an env var for overriding it. No need for hard git stuff. I use upstream

**Review by Schniz — COMMENTED** [review]
So it doesn't update, right? Only notified that there is an update

**Comment by deanshub** [comment]
> So it doesn't update, right? Only notified that there is an update

yes, I want to move in steps, let's see if this works well before we auto-update

**Pull request merged**